font/src/ft/mod.rs

```diff
@@ -54,23 +54,23 @@ impl ::Rasterize for FreeTypeRasterizer {
         })
     }
 
-    fn metrics(&self, key: FontKey, size: Size) -> Result<Metrics, Error> {
+    fn metrics(&self, key: FontKey) -> Result<Metrics, Error> {
         let face = self.faces
             .get(&key)
             .ok_or(Error::FontNotLoaded)?;
 
-        let scale_size = self.dpr as f64 * size.as_f32_pts() as f64;
+        // A face should always have a size and size metrics
+        let size_metrics = match face.size_metrics() {
+            Some(metrics) => metrics,
+            None => panic!("There was an error loading font size metrics"),
```

Comment:
The `metrics` method returns a `Result`. Let's take advantage of that instead of panicking.